src-db/database/model/tables/ETDEP_DEPENDENCY.xml

```diff
@@ -73,6 +73,10 @@
         <default><![CDATA[N]]></default>
         <onCreateDefault><![CDATA[N]]></onCreateDefault>
       </column>
+      <column name="DELETEDEPENDENCY" primaryKey="false" required="true" type="CHAR" size="1" autoIncrement="false">
```

Comment:
Add a check constraint in the table, to restrict possible values of this column to only 'Y' or 'N'

```
<check name="ETDEP_CHK_DELDEP"><![CDATA[DELETEDEPENDENCY IN ('Y', 'N')]]></check>
```